api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -286,50 +287,132 @@ private void updateInstitutionalAffiliations(Profile updatedProfile, DbUser user
     }
   }
 
-  private void validateUpdatedProfile(Profile updatedProfile, Profile prevProfile)
-      throws BadRequestException {
-    validateAndCleanProfile(updatedProfile);
-    if (StringUtils.isEmpty(updatedProfile.getAreaOfResearch())) {
-      throw new BadRequestException("Research background cannot be empty");
+  private void validateStringLength(String field, String fieldName, int max, int min) {
+    if (field == null) {
+      throw new BadRequestException(String.format("%s cannot be left blank!", fieldName));
+    }
+    if (field.length() > max) {
+      throw new BadRequestException(
+          String.format("%s length exceeds character limit. (%d)", fieldName, max));
+    }
+    if (field.length() < min) {
+      if (min == 1) {
+        throw new BadRequestException(String.format("%s cannot be left blank.", fieldName));
+      } else {
+        throw new BadRequestException(
+            String.format("%s is under character minimum. (%d)", fieldName, min));
+      }
+    }
+  }
+
```

Comment:
Can we just add validation methods to the ProfileTargetProperty enum and cycle through those instead of putting a bunch of if's here? That way we won't have to remember to update them all.

Where we have
```
enum class ProfileTargetProperty
constructor(override val propertyName: String, override val extractor: (Profile) -> String?) : ModelBackedTargetProperty<Profile>
```
we can just add another parameter like `String? -> boolean`

You could also add another interface for this like `ValidatingTargetProperty<Profile>`.

The downside is we have to make everything into a string.